scripts/flash/fill-market-orders.js

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var async = require("async");
+var chalk = require("chalk");
+var fillOrder = require("../dp/lib/fill-order");
+
+function fillMarketOrderInternal(augur, marketIds, orderType, outcomeId, universe, auth, callback) {
+  augur.markets.getMarkets({ universe: universe, sortBy: "endDate", isSortDescending: true }, function (err, marketIds) {
+    if (!marketIds || marketIds.length === 0) {
+      console.log(chalk.red("No markets available"));
+      callback("No Markets");
+    }
+    augur.markets.getMarketsInfo({ marketIds: marketIds }, function (err, marketInfos) {
+      if (err) {
+        console.log(chalk.red("Error "), chalk.red(err));
+        return callback(err);
+      }
+      if (!marketInfos || !Array.isArray(marketInfos) || !marketInfos.length) {
+        return callback("No Market Info");
+      }
+      var market = marketInfos[0];
+      var marketId = market.id;
+      console.log(chalk.yellow.dim("outcome:"), chalk.yellow(outcomeId), chalk.yellow.dim("order type:"), chalk.yellow(orderType));
+      console.log(chalk.green.dim("OrderId:"), chalk.green.dim("Account:"), chalk.green.dim("Amount:"), chalk.green.dim("FullPrecisionPrice:"), chalk.green.dim("FullPrecisionAmount"));
+      augur.trading.getOrders({ marketId: marketId, outcome: outcomeId, orderType: orderType }, function (err, orderBook) {
+        if (err) {
+          console.error(err);
+          callback(err);
+        }
+        if (!orderBook[marketId]) {
+          callback("No Market Orders Found");
+        }
+        var orders = orderBook[marketId][outcomeId][orderType];
+        async.eachSeries(Object.keys(orders), function (orderId, nextOrder) {
+          var order = orders[orderId];
+          console.log(chalk.yellow(order.fullPrecisionPrice), chalk.yellow(order.fullPrecisionAmount));
+          fillOrder(augur, universe, auth.address, outcomeId, order.fullPrecisionAmount, orderType, auth, function (err) {
+            if (err) {
+              nextOrder(err);
```

Comment:
You should `return nextError(err)` here, or else you'll call `nextOrder` again below